ui/desktop/src/recipe/recipeStorage.ts

```diff
@@ -0,0 +1,339 @@
+import { Recipe } from './index';
```

Comment:
Why did we decide to have a different format for recipes for the frontend than what we have for the cli? We now store SavedRecipe instead of recipe.

What I think we should do here, is do this through the server and not have this code directly write to disk, same as we do for sessions. that way we can make sure that the command line and the desktop have the same idea about where to look for recipes and the format for a recipe.

Can we fix this ASAP?